src/api/operations/form.fragments.graphql

```diff
@@ -121,10 +121,12 @@ fragment FormDefinitionMetadata on FormDefinition {
   role
   title
   cacheKey
+  identifier
 }
 
 fragment FormDefinitionFields on FormDefinition {
   ...FormDefinitionMetadata
+  rawDefinition
```

Comment:
Oh whoops, we don't want this here. This will case the raw JSON definition to be resolved alongside the parsed definition every time a form is loaded, which is unnecessary. Can you set up a different fragment that's used for the admin functions?